lusSTR/wrappers/snps_format.py

```diff
@@ -154,6 +155,8 @@ def process_sigprep_snps(foren_df, sampid, projid, analyid):
     data_list = []
     for j, row in foren_df.iterrows():
         snpid = foren_df.loc[j, "Locus"]
+        # if snpid == "N29insA" or snpid == "rs312262906_N29insA":
+        #    snpid = "rs312262906"
```

Comment:
Did you want to keep this commented-out code?

Reply:
Nope- good catch. Will remove!